### PR TITLE
Feature/update fluxes in evtgen

### PR DESCRIPTION
**Pull request by cg-laser**
adds additional flux options to event generator

### CODE REVIEW

**Review by shallmann — COMMENTED** [review]
Why did you remove the get_GZK_1 one? as far as I can see it is not imported from somewhere else... 

Maybe it would be a good point to collect fluxes somewhere more useful than in the generator and in the Sensitivities directory?
(Not sure if somebody had already started on this at some point? @lpyras?)

**Comment by cg-laser** [comment]
I actually replaced it with the corresponding function from `examples/Sensitivities/fluxesE2_3.py` ;-) 
Most neutrino fluxes are defined there, but as you say we should at some point restructure that and clearly define all neutrino fluxes in a separate utility function.

**Comment by cg-laser** [comment]
@sjoerd-bouma do you know why the documentation test fails? For me it looks good and I have a hard time finding the problem.

**Comment by sjoerd-bouma** [comment]
> @sjoerd-bouma do you know why the documentation test fails? For me it looks good and I have a hard time finding the problem.

This is the error message from the [github action](https://github.com/nu-radio/NuRadioMC/runs/6109840597?check_suite_focus=true):
```
[indentation]
/home/runner/work/NuRadioMC/NuRadioMC/NuRadioMC/EvtGen/generator.py:docstring of NuRadioMC.EvtGen.generator.get_energies:22: ERROR: Unexpected indentation.
/home/runner/work/NuRadioMC/NuRadioMC/NuRadioMC/EvtGen/generator.py:docstring of NuRadioMC.EvtGen.generator.get_energies:23: WARNING: Block quote ends without a blank line; unexpected unindent.
```

The problem is that the docstring for `NuRadioMC.EvtGen.generator.get_energies` contains a list which is not separated from the rest of the docstring by newlines, as required by `rst` syntax. See e.g. [here](https://nu-radio.github.io/NuRadioMC/Introduction/pages/contributing.html#writing-docstrings) and [here](https://nu-radio.github.io/NuRadioMC/Introduction/pages/contributing.html#lists) for some pointers in our documentation on how the rst syntax works.

The short version is: put an empty line before and after the list of fluxes back in lines 343 & 352.

**Comment by shallmann** [comment]
Ah ok, you are right. you replaced all the occurences. Maybe you can still adjust the docstrings (remove (see get_GZK_1 function for details))  that refer to the function that is now not there any more...

**Comment by cg-laser** [comment]
@shallmann I fixed the doc string

**Review by anelles — APPROVED** [review]
I think everything that has been discussed has been addressed and the rest of the changes look good to me.
